contrib/ansible/roles/objectstore/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Set facts
```

Comment:
These should be Ansible defaults (objectstore/defaults/main.yml)

Reply:
done